diff -urN SFML-1.6/include/SFML/System/Resource.hpp SFML-1.6-patched/include/SFML/System/Resource.hpp
--- SFML-1.6/include/SFML/System/Resource.hpp	2010-01-27 23:53:54.000000000 +1000
+++ SFML-1.6-patched/include/SFML/System/Resource.hpp	2012-01-13 20:14:43.000000000 +1100
@@ -29,6 +29,7 @@
 // Headers
 ////////////////////////////////////////////////////////////
 #include <set>
+#include <stdlib.h>
 
 
 namespace sf
diff -urN SFML-1.6/src/SFML/Graphics/Image.cpp SFML-1.6-patched/src/SFML/Graphics/Image.cpp
--- SFML-1.6/src/SFML/Graphics/Image.cpp	2010-01-28 00:00:05.000000000 +1000
+++ SFML-1.6-patched/src/SFML/Graphics/Image.cpp	2012-01-13 20:23:03.394254128 +1100
@@ -33,6 +33,7 @@
 #include <iostream>
 #include <vector>
 #include <string.h>
+#include <memory.h>
 
 
 namespace sf
