diff -up SFML-2.5.0/CMakeLists.txt.omv~ SFML-2.5.0/CMakeLists.txt
diff -up SFML-2.5.0/src/SFML/Window/CMakeLists.txt.omv~ SFML-2.5.0/src/SFML/Window/CMakeLists.txt
--- SFML-2.5.0/src/SFML/Window/CMakeLists.txt.omv~	2018-08-05 21:07:32.634648547 +0200
+++ SFML-2.5.0/src/SFML/Window/CMakeLists.txt	2018-08-05 21:07:46.634731260 +0200
@@ -262,7 +262,7 @@ endif()
 
 if(SFML_OS_LINUX)
     sfml_find_package(UDev INCLUDE "UDEV_INCLUDE_DIR" LINK "UDEV_LIBRARIES")
-    target_link_libraries(sfml-window PRIVATE UDev)
+    target_link_libraries(sfml-window PRIVATE UDev pthread)
 elseif(SFML_OS_WINDOWS)
     target_link_libraries(sfml-window PRIVATE winmm gdi32)
 elseif(SFML_OS_FREEBSD)
